server/dotnet/Program.cs

```diff
@@ -0,0 +1,21 @@
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Hosting;
+
+namespace sample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            CreateHostBuilder(args).Build().Run();
+        }
+
+        public static IHostBuilder CreateHostBuilder(string[] args) =>
+            Host.CreateDefaultBuilder(args)
+                .ConfigureWebHostDefaults(webBuilder =>
+                {
+                    webBuilder.UseStartup<Startup>();
+                    webBuilder.UseWebRoot("../../client/");

```

Comment:
Can we add this as an environment variable?

Reply:
+1, our .env.example has it set as `STATIC_DIR`

Reply:
Let's also add a comment that this is so we can reuse the same client code across different server languages and not a requirement to get it to work with Stripe

Reply:
Added reading the web root path from `STATIC_DIR` and additional comment to explain this is only for sample purposes